tasks/62_swap-cases/swap-cases.js

```diff
@@ -0,0 +1,22 @@
+
+
+export function swapCases(str) {
```

Comment:
Try solving using toUpperCase() to find out if letter is upper and switch according to that.